requirements.txt

```diff
@@ -32,4 +32,6 @@ boto3~=1.16.54
 
 django-ses>=2.0.0 , <2.1.0
 
-openlxp-notification >=1.2.0
\ No newline at end of file
+openlxp-notification >=1.2.0
+
+openlxp-authentication>=0.1, <=1.0
```

Comment:
Nit: missing space character between package name and version specifier (I think)